conda_smithy/templates/azure-pipelines-win.yml.tmpl

```diff
@@ -56,7 +56,10 @@ jobs:
 
     - task: CondaEnvironment@1
       inputs:
-        packageSpecs: 'python=3.6 conda-build conda conda-forge::conda-forge-ci-setup=2' # Optional
+        # conda-build 3.17 and crytography seem to pull in different versions of openssl
+        # this causes problems in the build system
+        # therefore, specify that the higher conda-build version should take precedence
+        packageSpecs: 'python=3.6 conda-build>=3.17 conda conda-forge::conda-forge-ci-setup=2' # Optional
```

Comment:
You can add this condition to `conda-forge-ci-setup`

Reply:
I'm not so sure I can. I feel like it would just downgrade conda-forge-ci-setup, so that it still respects the 2.x pin, but has a higher version of openssl.